### PR TITLE
Reflector Trenchcoat replaces armory reflective vest

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Adds a reflective trenchcoat, that replaces the armor vest in the armory. The old one can still be bought from cargo. (Objectives will point to the trenchcoat now.)

## Why It's Good For The Game
I feel like my old PR didn't go far enough, same arguments apply. Ablative armor is really weak and exists only for the traitor objective. This makes it slightly more useful.

## Changelog
:cl:
balance: armory has a reflective trenchcoat
tweak: you can still get the vest from cargo but it's not the objective anymore 
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
why does github do this to me

**Comment by ATH1909** [comment]
Ah, and now you can't cheese the reflective vest theft objective by just buying a vest from cargo!

Good change, IMO.

**Comment by HomocidlePancakes** [comment]
7AD96D5C-136C-47FF-BB8F-1889BA4DE29C
Gee, I wonder why it’s conflicting

**Comment by Tlaltecuhtli** [comment]
it looks ugly

**Comment by carlarctg** [comment]
> it looks ugly

make it better

**Comment by necromanceranne** [comment]
When you wanna look like you're out of Tron

**Comment by WaterStBlues** [comment]
Remember to modify the traitor goal so getting the objective doesn't require a fucking cargo order

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -617,6 +617,54 @@
...
+	strip_delay = 30
+	equip_delay_other = 40
+	var/hit_reflect_chance = 50
+
+/obj/item/clothing/suit/hooded/ablative/Initialize()
+	. = ..()
+	allowed = GLOB.security_vest_allowed
+
+/obj/item/clothing/suit/hooded/ablative/IsReflect(def_zone)
+	if(!(def_zone in list(BODY_ZONE_CHEST, BODY_ZONE_PRECISE_GROIN, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))) //If not shot where ablative is covering you, you don't get the reflection bonus!
```
Comment (Fikou) [inline code/modules/clothing/suits/miscellaneous.dm]: why not if(def_zone in list(BODY_ZONE_HEAD etc

Comment (nemvar) [reply code/modules/clothing/suits/miscellaneous.dm]: because that would need updating should new zones ever get added.

**Comment by Shapsy** [comment]
A reflective trenchcoat is probably one of the most absurd and funny ideas I've seen in SS13

File: code/modules/clothing/suits/miscellaneous.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/clothing/suits/miscellaneous.dm]: because the hood is a separate item? what

Comment (carlarctg) [reply code/modules/clothing/suits/miscellaneous.dm]: oh

**Comment by PKPenguin321** [comment]
me when im laser immune
image

**Comment by Shapsy** [comment]
Cosplaying as an SS officer at the discotheque

**Comment by Learner10** [comment]
It does look like it'd be from Tron. And honestly, I'm quite fine with that.

**Comment by TooFewSecrets** [comment]
Support if you can get it to actually pass the tests, an antag objective being available for 2200 credits is stupid.

**Comment by WaterStBlues** [comment]
Fun idea: shooting a laser pointer at someone wearing this creates a bunch of scattered laser pointer lights around the wearer.
Ignores eye protection. Warden charging at you while wearing this? Laser pointer him to blind the two sec goons he has with him.

**Comment by anconfuzedrock** [comment]
Shadowflame yet again being shadowflame
Lavaland is an arbitrary luck based exception that shouldn't exist and is uninteresting
plasma tank is a joke objective
why are you relating kill objs to this

counterpoint who the hell cares shadowflame most objs are single version anyways its a traitor obj
I'm glad this ugly fucker doesn't replace the regular vest but I'm glad this ugly fucker exists

**Comment by Shadowflame909** [comment]
Counterpoint: I care.

**Comment by necromanceranne** [comment]
Countercounter point who fucking cares if your objective is to steal the captain's bath towel. It's a goal and that's all that matters. How you achieve that goal is entirely up to you and anything in the game is fair play. Sometimes it's easy. Sometimes it is dead hard.

**Comment by Learner10** [comment]
Random Interjection: Why isn't stealing the captain's bath towel a potential objective? You could kidnap the captain, hand him a random towel, brainwash him to make him use it, then do whatever with the captain after.

**Comment by lolman360** [comment]
interdiction: port goon objectives/miscreants when

**Review by 81Denton — CHANGES_REQUESTED** [review]
Remove merge conflicts from the premises
Also if you have time, please post some ingame pictures so we can tell if it looks like a disco SS uniform when worn by someone.

**Comment by carlarctg** [comment]
well i can tell you that it looks like shit

**Comment by carlarctg** [comment]
is probably mergeable now

**Pull request merged**